raiden/tests/scenarios/mfee1_flat_fee.yaml

```diff
@@ -65,12 +65,12 @@ scenario:
                 expected_routes:
                   - [0, 1, 2, 3]
 
-            # 200 TKN fees are calculated, 5% margin of that is 10 TKN
-            - assert: {from: 0, to: 1, total_deposit: 100_000, balance: 89_790, state: "opened"}
-            - assert: {from: 1, to: 0, total_deposit: 0, balance: 10_210, state: "opened"}
+            # 200 TKN fees are calculated and margin of 200 * 3% + 10000 * 0.05% = 11
```

Comment:
where do the terms for this margin calculation come from? `200 * 3% + 10000 * 0.05% = 11`

Reply:
The margin is defined as `fee * 3% + amount * 0.05%`. I didn't want to get too elaborate. Should I add that explanation?

Reply:
Maybe you can add a placeholder with a link to the explanation in the spec?

Reply:
A self-contained explanation in the scenario would indeed be cool. It's nice to just read the scenario and understand what's happening and what is being tested.